test/governance-relay.test.ts

```diff
@@ -41,7 +41,7 @@ describe("Governance", async function () {
   let l2user: Account;
 
   let l2GovRelay: StarknetContract;
-  let l2Spell: StarknetContract;
+  let l2Spell: string;
```

Comment:
could we rename this variable to `l2SpellHash` to emphasize the fact that it's a class hash?